.claude/skills/radio-link-testing/scripts/tier1-patch.sh

```
@@ -0,0 +1,94 @@
+now(){ date +%s; }
+
+# --- split locally into N parts, record expected CRCs ---
+split -d -n "$N" "$SRC" "$WORK/p_" || exit 1
+parts=( "$WORK"/p_* )
+echo "[tier1] $SRC -> ${#parts[@]} parts ($(wc -c <"$SRC") bytes); whole F'CRC=$(fcrc "$SRC")"
+declare -a EXP DEST
+for i in "${!parts[@]}"; do EXP[$i]=$(fcrc "${parts[$i]}"); DEST[$i]="//tp_${i}.bin"; done
+
+# --- upload + verify a single part; sets global VERIFY_OK ---
+upload_part(){ # idx  -- kick uplink in bg (await never returns in TX-off mode), poll console
+  local i="$1" dst="${DEST[$i]}" src="${parts[$i]}" mark t0 pid s ok=0
```

Shellcheck (0.11.0)

[warning] This assignment is used again in this 'local', but won't have taken effect. Use two 'local's.

(SC2318)